meuapp_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative 'meuapp'
+require 'rack/test'
+
+set :environment, :test
+
+describe 'Server Service' do
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application
+  end
+
+  it 'should load the home page' do
```

Comment:
Lembre-se que por padrão no rspec não documentamos testes com `should`. As mensagens devem ser assertivas.